File: src/main/java/green/seagull/very/good/purchase/repository/PurchaseCrudRepository.java
package green.seagull.very.good.purchase.repository;

import green.seagull.very.good.purchase.domain.Purchase;
import org.springframework.data.repository.reactive.ReactiveCrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface PurchaseCrudRepository extends ReactiveCrudRepository<Purchase, String> {

}

File: src/main/kotlin/green/seagull/very/good/purchase/repository/PurchaseCrudRepository.kt
package green.seagull.very.good.purchase.repository

import green.seagull.very.good.purchase.domain.Purchase
import org.springframework.data.repository.reactive.ReactiveCrudRepository
import org.springframework.stereotype.Repository

@Repository
interface PurchaseCrudRepository : ReactiveCrudRepository<Purchase, String>